wagtail/wagtailimages/templates/wagtailimages/images/index.html

```diff
@@ -1,25 +1,79 @@
 {% extends "wagtailadmin/base.html" %}
-{% load wagtailimages_tags %}
-{% load i18n %}
+{% load wagtailimages_tags i18n compress static %}
 
 {% block titletag %}{% trans "Images" %}{% endblock %}
 {% block bodyclass %}menu-images{% endblock %}
 {% block extra_js %}
-    <script>
-        window.headerSearch = {
```

Comment:
It's still there, further down the block. However, I see it's now inside a `{% compress js %}` block, which is something we definitely want to avoid, as it contains dynamic values passed in from the Django template (which will interfere with offline compression in particular).